src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -113,10 +125,38 @@ private void setAccelerator(MenuItem menuItem, KeyCombination keyCombination) {
     void fillInnerParts() {
         personListPanel = new PersonListPanel(logic.getFilteredPersonList());
         appointmentListPanel = new AppointmentListPanel(logic.getFilteredAppointmentList());
-        listPanelPlaceholder.getChildren().add(personListPanel.getRoot());
+        personVBox.getChildren().addAll(personLabel, personListPanel.getRoot());
+        appointmentVBox.getChildren().addAll(appointmentLabel, appointmentListPanel.getRoot());
+
+
+        hbox.setSpacing(16);
+        hbox.setPadding(new Insets(15, 12, 15, 12));
+        hbox.setAlignment(Pos.CENTER);
+        HBox.setHgrow(personVBox, Priority.ALWAYS);
+        HBox.setHgrow(appointmentVBox, Priority.ALWAYS);
+
+        listPanelPlaceholder.getChildren().add(hbox);
+        hbox.setMinWidth(listPanelPlaceholder.getWidth());
+
+
+        personLabel.setStyle("-fx-font-weight: bold;");
+        appointmentLabel.setStyle("-fx-font-weight: bold;");
+        personLabel.setStyle("-fx-text-fill: #3c3c3c; -fx-font-weight: bold;");
+        appointmentLabel.setStyle("-fx-text-fill: #3c3c3c; -fx-font-weight: bold;");
+        hbox.setStyle("-fx-background-color: #f2e1b3;");
+        commandBoxPlaceholder.setStyle("-fx-min-height: 65px");
+
+        personVBox.setSpacing(5);
+        appointmentVBox.setSpacing(5);
+
+        personVBox.setMaxWidth(Double.MAX_VALUE);
+        personVBox.setPrefWidth(Region.USE_COMPUTED_SIZE);
+        appointmentVBox.setMaxWidth(Double.MAX_VALUE);
+        appointmentVBox.setPrefWidth(Region.USE_COMPUTED_SIZE);
```

Comment:
I also noticed that the two boxes for the person and appointment list do not fit the height.

<img width="740" alt="Screenshot 2024-10-30 at 3 58 38 AM" src="https://github.com/user-attachments/assets/1d97be44-ad72-4b5b-904d-34137e4547eb">

Do you think we can extend the list till the bottom so that we maximise the space here?